docs/website/data/integrations.yaml

```diff
@@ -1,6 +1,46 @@
 # This file is used to generate the OPA ecosystem page: https://www.openpolicyagent.org/docs/latest/ecosystem/
 # For instructions on how to add an entry, see https://github.com/open-policy-agent/opa/tree/main/docs#integration-index
 integrations:
+
+  Torque:
+     title: Torque
+     description: Torque by Quali is a cloud-based platform that provides infrastructure automation and orchestration solutions for digital transformation and DevOps initiatives. Troque utilizes Open Policy Agent (OPA) to enforce policy-as-code, enabling users to define and automate their own security, compliance, and governance policies across their infrastructure.
+     labels:
+       category: Environments as a Service
+       layer: Infrastructure
+       type: poweredbyopa
+     software:
+       - terraform
+       - helm
+       - cloudformation
+       - kubernetes
+       - ansible
+       - aws
+       - gcp
+       - azure
+     tutorials:
+       - https://docs.qtorque.io/admin-guide/governance/policies
+     code:
+       - https://github.com/QualiTorque/opa
+     inventors:
+       - Quali
```

Comment:
Should be lowercase, and reference an organization found further down in the file.